src/app/components/pages/admin-cell/admin-cell.component.html

```diff
@@ -70,3 +80,88 @@ <h2>Cellule de {{ cell.name }}</h2>
     </tr>
   </table>
 </div>
+
+
+<app-my-modal name="create event"
+              title="Création d'un event"
+              button2Label="Créer"
+              (button1)="eventForm.reset()"
+              (button2)="createEvent()"
+              typeModal="form"
+              button2Style="button--success">
+
+  <form [formGroup]="eventForm">
+
+    <input type="hidden" formControlName="cell_id">
+
+    <div class="form-group">
+      <label for="nb_volunteers_needed" class="form-label">Nombre de volontaires requis:</label>
+      <input class="input" formControlName="nb_volunteers_needed" id="nb_volunteers_needed" type="number" autofocus>
+      <div *ngIf="eventForm.controls['nb_volunteers_needed'].hasError('required')"
+           class="form-text form-text--right form-text--danger">Le nombre de volontaires est obligatoire.
+      </div>
+      <div *ngIf="eventForm.controls['nb_volunteers_needed'].hasError('min')"
+           class="form-text form-text--right form-text--danger">Un nombre negatif de volontaires n'est point possible.
```

Comment:
Accent on `negatif`, maybe can we add a `min=0` on the input to help people not have this error ? (so only developer who play with the HTML can see this error)